frontend/packages/local-storage-operator-plugin/integration-tests/tests/disks-list-page.scenario.ts

```diff
@@ -0,0 +1,87 @@
+import { $, ExpectedConditions as until, browser, Key } from 'protractor';
+import { execSync } from 'child_process';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { resourceRows, isLoaded } from '@console/internal-integration-tests/views/crud.view';
+import { clickNavLink } from '@console/internal-integration-tests/views/sidenav.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import {
+  page,
+  diskList,
+  clickFilterDropdown,
+  checkDiskFilter,
+  clearDiskFilter,
+} from '../views/disks-list-page.views';
+import { LOCAL_STORAGE_NAMESPACE } from '../../src/constants/index';
+import {
+  LocalVolumeDiscoveryResultKind,
+  DiskMetadata,
+  DiskStates,
+} from '../../src/components/disks-list/types';
+
+// Prerequisite: Local volume discovery to be present
+describe('Disk list is accessible from Nodes view', () => {
+  let discoveredDevices: DiskMetadata[];
+  beforeAll(async () => {
+    const lvdrJson = JSON.parse(
+      execSync(
+        `kubectl get localvolumediscoveryresults -n ${LOCAL_STORAGE_NAMESPACE} -o json`,
+      ).toString(),
+    );
+    const lvdr: LocalVolumeDiscoveryResultKind = lvdrJson.items[0];
+    const { nodeName } = lvdr.spec;
+    discoveredDevices = lvdr.status.discoveredDevices;
+    await browser.get(`${appHost}/`);
+    await clickNavLink(['Compute', 'Nodes']);
+    await isLoaded();
+    click($(`a[data-test-id="${nodeName}"]`));
```

Comment:
You can create a function and pass this dynamic value in view.ts file. 
It will be useful, if the selectors get changed in future version.

Reply:
https://github.com/openshift/console/pull/6606#discussion_r487679934